src/Bridges/Constraint/zero_one.jl

```diff
@@ -74,32 +69,12 @@ end
 
 function MOI.get(
     model::MOI.ModelLike,
-    attr::Union{MOI.ConstraintPrimal,MOI.ConstraintPrimalStart},
+    attr::Union{MOI.ConstraintPrimal},
     bridge::ZeroOneBridge,
 )
     return MOI.get(model, attr, bridge.interval_index)
 end
 
-function MOI.supports(
-    ::MOI.ModelLike,
-    ::MOI.ConstraintPrimalStart,
-    ::Type{<:ZeroOneBridge},
-)
-    return true
-end
-
-function MOI.set(
-    model::MOI.ModelLike,
-    attr::MOI.ConstraintPrimalStart,
-    bridge::ZeroOneBridge{T},
-    value,
-) where {T}
-    MOI.set(model, attr, bridge.integer_index, value)
-    MOI.set(model, attr, bridge.interval_index, value)
```

Comment:
@blegat does `ConstraintPrimalStart` make sense for a `ZeroOne` constraint?

Reply:
Yes, a ConstraintPrimalStart is the starting value of the function. It's well-defined for all constraints

Reply:
Sure. But does any solver actually implement it...

Reply:
Since most solvers only support VariableIndex-in-Integer, and setting the starting value of this is a bit redundant, I don't think so. However, if a solver want to implemented ScalarAffineFunction in Integer, they will need this.

Reply:
To rephrase: in what situation would a solver benefit from being able to set `ConstraintPrimalStart` of a `ZeroOne` constraint.